tutorials/parquet-catalog-demos/neowise-source-table-lightcurves.md

```diff
@@ -11,6 +11,11 @@ kernelspec:
   name: python3
 ---
 
+An executed version of this notebook can be seen at
+[https://irsa.ipac.caltech.edu/docs/notebooks/neowise-source-table-lightcurves.html](https://irsa.ipac.caltech.edu/docs/notebooks/neowise-source-table-lightcurves.html).
```

Comment:
Use a narrative text here for the URL, that is way more reader (and screen-reader) friendly. One suggestion, but anything else works, really.
```suggestion
[IRSA website]https://irsa.ipac.caltech.edu/docs/notebooks/neowise-source-table-lightcurves.html).
```